dev_tools/packaging/produce-package.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
```

Comment:
This line shouldn't be here

Reply:
Why not? These are bash scripts so it's appropriate to start off with a shebang.